api/proc.capnp

```diff
@@ -0,0 +1,42 @@
+using Go = import "/go.capnp";
+
+@0xd78885a0de56b292;
+
+$Go.package("proc");
+$Go.import("github.com/wetware/ww/internal/api/proc");
+
+interface Executor(T) {
+    exec @0 (command :T) -> (proc :Process);
+}
+
+interface Process {
+    start @0 () -> ();
```

Comment:
I have been thinking about it. I do think we should leave as it is. That is to say, I think is a REALLY good idea to have two separate methods `exec` and `start`. Why is so? Because this way we can decouple the configuration/definition of a process and its execution. If we remove the `start` call, then we do not let applications/orchestrators to delegate the execution of the process to others. Right now, I cannot come up with a specific example, but I think in the future we will have a use case where an application creates a process, stores in an anchor, and another application finds it an executes it.

Reply:
Agreed.  Leaving this open since we're still thinking about it.